components/button/button-add.js

```diff
@@ -56,49 +74,28 @@ class ButtonAdd extends PropertyRequiredMixin(FocusMixin(LocalizeCoreElement(Lit
 				width: 100%;
 			}
 			button:hover .line,
-			button:${unsafeCSS(getFocusPseudoClass())} .line {
+			button:focus .line {
```

Comment:
I switched this to use `focus` rather than `focus-visible` because VisibleOnAncestorMixin uses `focus` and having some parts use `focus` and others use `focus-visible` resulted in some strange focus behaviour.